### ISSUE TITLE
Streamdeck companion button

### ISSUE THREAD

**Issue by brianteeman** [bitfocus/companion-satellite#29]
Firstly thanks for the awesome work - it's really made remote working a breeze.

Currently I am using satellite natively on my streamdeck but I would really like to be able to use the streamdeck companion button instead so that I can use other features of the streamdeck.

However the streamdeck companion button only finds local instances of companion. Am I missing something to make the streamdeck see the satellite or is it just not possible

**Comment by Julusian** [comment]
This isnt possible currently.

**Comment by brianteeman** [comment]
would you consider adding it to the roadmap

**Comment by davisa6477** [comment]
Sorry I didn't see this before posting my issue. This is something I would like to see as well.

**Comment by Julusian** [comment]
I dont think this is relevant anymore, as the plugin has supported connecting remotely to companion for a couple of releases now
